### PR TITLE
[CGE][#1] Page 1 with Searchbar

### PR DESCRIPTION
**Pull request by CorentinAuTibet**
The first page with the autocomplete search bar and 2 buttons. 1 deleting button and a search button to go on page 2

### CODE REVIEW

File: src/app/pages/secondary-page/secondary-page.component.html
Comment (TheDestiny45) [inline src/app/pages/secondary-page/secondary-page.component.html:1]: Rename 'secondary-page' to 'result-page' ?

File: src/app/shared-component/keyword.ts
Comment (TheDestiny45) [inline src/app/shared-component/keyword.ts:1]: To delete

File: src/styles.scss [hunk 1]
```diff
@@ -1 +1,7 @@
 /* You can add global styles to this file, and also import other style files */
+
+html, body { height: 100%; }
+body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+
+html, body { height: 100%; }
+body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
```
Comment (TheDestiny45) [inline src/styles.scss]:
```suggestion
html, body { height: 100%; }
body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
```

**Review by TheDestiny45 — CHANGES_REQUESTED** [review]
Small changes need to be taken into account

File: src/app/pages/home-page/home-page.component.ts
Comment (TheoSouchon) [inline src/app/pages/home-page/home-page.component.ts:1]: import unused

**Review by TheDestiny45 — APPROVED** [review]

**Review by baptiste1403 — APPROVED** [review]

**Pull request merged**